src/mumble/GlobalShortcut.ui

```diff
@@ -216,6 +216,16 @@
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
+          <widget class="QCheckBox" name="qcbEnableUIAccess">
+           <property name="whatsThis">
+            <string>&lt;b&gt;Enable Windows UI automation&lt;/b&gt;.&lt;br /&gt;Also known as &quot;UIAccess&quot;. This allows Mumble to recieve global shortcut events from programs running at high privilege levels.</string>
+           </property>
+           <property name="text">
+            <string>Enable Windows UI automation</string>
```

Comment:
I don't really like this short description. Why not call it something more prescriptive like "Enable shortcuts in priviledged applications", "Allow access to priviledge applications" or something like that instead? Hardly any programmer even knowns about ```UIAccess```.